tests/ControversialTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+require_once 'src/Entities/Blog.php';
+
+class ControversialTest extends TestCase
```

Comment:
Tests look good, but since they are testing the BlogEntity, can you just move these methods into BlogTest.php and remove this file